x/mint/test/mint_test.go

```diff
@@ -0,0 +1,148 @@
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	minttypes "github.com/celestiaorg/celestia-app/x/mint/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/rpc/client"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cctx testnode.Context
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	if testing.Short() {
+		s.T().Skip("skipping mint integration test in short mode.")
+	}
```

Comment:
[not blocking]
I forget what we do in other tests, but imo we should stick this in the actual function calling the test and not the setup

Reply:
I copied from other integ tests but I agree it makes sense to move it to the function calling test.

Ref: https://github.com/rootulp/celestia-app/blob/8e448efe9641150e170e99b52748533cdf338c4f/x/blob/client/testutil/integration_test.go#L45-L47

Verified test time is similar


**Before**
```
$ go test ./... -short -count=1
?   	github.com/celestiaorg/celestia-app/x/mint/client/cli	[no test files]
ok  	github.com/celestiaorg/celestia-app/x/mint	0.517s
ok  	github.com/celestiaorg/celestia-app/x/mint/client/testutil	16.855s
ok  	github.com/celestiaorg/celestia-app/x/mint/keeper	1.466s
ok  	github.com/celestiaorg/celestia-app/x/mint/simulation	0.894s
ok  	github.com/celestiaorg/celestia-app/x/mint/test	1.787s
ok  	github.com/celestiaorg/celestia-app/x/mint/types	2.641s
```

**After**
```
$ go test ./... -short
?   	github.com/celestiaorg/celestia-app/x/mint/client/cli	[no test files]
ok  	github.com/celestiaorg/celestia-app/x/mint	0.515s
ok  	github.com/celestiaorg/celestia-app/x/mint/client/testutil	16.903s
ok  	github.com/celestiaorg/celestia-app/x/mint/keeper	1.203s
ok  	github.com/celestiaorg/celestia-app/x/mint/simulation	1.406s
ok  	github.com/celestiaorg/celestia-app/x/mint/test	1.801s
ok  	github.com/celestiaorg/celestia-app/x/mint/types	2.389s
```

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1791/commits/078fd22393f9d0d74f23653e19a1bb6f6445e39f